### DISCUSSION TITLE
1.2.0 Features so far...

### DISCUSSION THREAD

**Discussion by sproket** [sproket/Persism#21] [General]
* Added support for Views
* Added query 'select *' method to session
* Finalized support for Informix (Tests added)
* Changed query statements to make explicit forward only and read only
* Improve readRecord performance
* Added BIT column to byte property conversion 

Also I have an "upsert" method but I'm not 100% convinced it's a good idea. Can anyone try the version in the experimental branch?

V2.0.0 is also coming along. I'll put another thread for that one.

I'm tagging watchers

@IdelsTak @Avinashachu007 @mring33621 @ericmm @Ozin @cyang-el @raytracer @sriio @aledell @mojo2012 @felixfoertsch @sirinath @aegershman @nezda @dstango @phlppnhllngr @CrommVardek @ngencic @zteater @IsSkyfalls @borsch @orange451 @Ali-RS @markscottwright @pablogrisafi1975  @g-gar @grossjonas @dstango  @pablogrisafi1975

**Comment by felixfoertsch** [comment]
Why are you tagging watchers? I'll be unstarring for this. 👎

**Reply by sproket** [reply]
I'm looking for some feedback. I was assuming you might be interested in the development. I won't tag you again.
